modules/module/GenerateLossesModel.py

```diff
@@ -83,24 +84,24 @@ def start(self):
         filename_loss_list: list[tuple[str, float]] = []
         # Don't really need a backward pass here, so we can make the calculation MUCH faster.
         with torch.inference_mode():
-            for epoch_step, batch in enumerate(step_tqdm):
-                    model_output_data = self.model_setup.predict(
-                        self.model,
-                        batch,
-                        self.config,
-                        self.model.train_progress,
-                        deterministic=True,
-                    )
-                    loss = self.model_setup.calculate_loss(
-                        self.model,
-                        batch,
-                        model_output_data,
-                        self.config,
-                    )
-                    filename_loss_list.append((batch['image_path'][0], float(loss)))
+            for _epoch_step, batch in enumerate(step_tqdm):
```

Comment:
isn't the `_something` naming reserved for protected class variables? using it here looks strange.

Reply:
Fixed! The linter hid the variable since it wasn't used, but I had to manually remove it since it wasn't needed.